routes/reservations.js

```diff
@@ -0,0 +1,50 @@
+const mongoose = require('mongoose')
+const express = require('express')
+const { Reservation, validate } = require('../schemas/reservationSchema')
+const router = express.Router()
+
+// TODO: add auth middlewear, when it will be ready
+
+router.get('/', async (req, res) => {
+  const reservations = await Reservation.find()
+
+  res.send(reservations)
+})
+
+router.post('/', async (req, res) => {
+  const { error } = validate(req.body)
+  if (error) return res.status(400).send(error.details[0].message)
+
+  let reservation = new Reservation({
+    userId: req.body.userId,
+    hotelId: req.body.hotelId,
+    roomId: req.body.roomId,
+    startDate: req.body.startDate,
+    endDate: req.body.endDate,
+    people: {
+      adults: req.body.adults,
+      children: req.body.children,
+    },
+  })
+
+  reservation = await reservation.save()
+
+  res.send(reservation)
+})
+
+router.put('/payment/:id', async (req, res) => {
+  const reservation = await Reservation.findByIdAndUpdate(
+    req.params.id,
+    {
+      isPaid: req.body.isPaid,
```

Comment:
you can actually set isPaid to true, we dont need it in body because we know that if this request comes client paid for its reservation